cosmic/tests/data/Params.ini

```diff
@@ -351,6 +351,17 @@ eddfac=1.0
 ; default=-2
 gamma=-2.0
 
+; don_lim is a flag which determines how much mass is lost during thermal timescale MT
+; don_lim = 0: assumes standard BSE choice as outlined in Hurley+2002
+; don_lim = -1: Follows Claeys+2014
+don_lim = 0
+
+; acc_lim is a flag which determines how much mass is accreted from the donor
+; if acc_lim > 0: this provides the fraction of mass accreted
+; acc_lim = 0: assumes standard BSE choice as outlined in Hurley+2002
+; acc_lim = -1: assumes the accretion is limited by the ratio of the KH timescales: tkh_acc/tkh_don
+; acc_lim = -2: assumes the accretion is limited by 10x's the ratio of the KH timescales: 10*tkh_acc/tkh_don
+acc_lim = -1
```

Comment:
Just making sure that you meant to set `acc_lim = -1` here, since you're using `acc_lim=0` in `test_evolve.py` below.